main.go

```diff
@@ -2,25 +2,11 @@
 package main
 
 import (
-	"flag"
 	"fmt"
-	"github.com/shopspring/decimal"
-	"volume_calculator/calculator"
+	"volume_calculator/cli"
 )
 
-func calculate(){
-	var shape string
-	flag.StringVar(&shape, "shape", "string", "a string")
-	radius := flag.Float64("radius", -1.0, "a float64")
-	length := flag.Float64("length", -1.0, "a float64")
-	width := flag.Float64("width", -1.0, "a float64")
-	hieght := flag.Float64("hieght", -1.0, "a float64")
-	flag.Parse()
-	calc := calculator.NewCalc(shape, decimal.NewFromFloat(*radius), decimal.NewFromFloat(*length),
-		decimal.NewFromFloat(*width), decimal.NewFromFloat(*hieght))
-	fmt.Println(calc.Volume())
-}
-
 func main() {
-	calculate()
+	calc := cli.Handle()
```

Comment:
here we have to load params from cli package and then pass them to the business logic layer.

```
cliParams := cli.Load()

// using DI pattern here
volume := calculator.Calculate((cliParams.shape, cliParams.radius, cliParams.length, cliParams.width, cliParams.hieght)
fmt.Println(volume)
```

on the business logic layer we need to call builder (Builder pattern) to build necessary shape: 
```
shape := shapes.NewShape(<params>)
volume := shape.CalculateVolume()
return volume
```